tests/ga/test_cgroupconfigurator.py

```diff
@@ -176,6 +172,43 @@ def test_initialize_should_not_enable_cgroups_when_the_agent_is_not_in_the_syste
             self.assertEqual(len(tracked), 0, "No cgroups should be tracked. Tracked: {0}".format(tracked))
             self.assertFalse(os.path.exists(agent_drop_in_file_cpu_quota), "{0} should not have been created".format(agent_drop_in_file_cpu_quota))
 
+    def test_initialize_should_not_enable_cgroups_v2(self):
```

Comment:
```suggestion
    def test_initialize_should_enable_cgroups_v2(self):
```
should enable cgroups v2?